app/src/main/res/layout/state_button_item.xml

```diff
@@ -14,44 +14,52 @@
     android:id="@+id/button_container_relative_layout"
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
-    android:layout_marginTop="16dp"
     android:gravity="center_vertical"
-    android:padding="16dp">
+    android:paddingStart="24dp"
+    android:paddingTop="12dp"
+    android:paddingEnd="24dp"
+    android:paddingBottom="136dp">
 
-    <ImageView
+    <ImageButton
       android:id="@+id/previous_state_image_view"
       android:layout_width="wrap_content"
       android:layout_height="wrap_content"
       android:layout_alignParentStart="true"
+      android:layout_margin="4dp"
       android:background="@drawable/previous_next_state_image_view_background"
       android:contentDescription="@string/previous_state_description"
-      android:padding="8dp"
+      android:elevation="2dp"
```

Comment:
This elevation works above api 21. Please check.

Reply:
I have mentioned that in High-Fidelity Mock Known issues. And also, this same case is appearing in nested recyclerview too, and therefore needs discussion with Ben.